test/unit/modules/EmailRecoveryModule/processRecovery.t.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { ModuleKitHelpers } from "modulekit/ModuleKit.sol";
+import { EmailRecoveryModuleBase } from "./EmailRecoveryModuleBase.t.sol";
+import { MODULE_TYPE_EXECUTOR } from "modulekit/accounts/common/interfaces/IERC7579Module.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { CommandHandlerType } from "../../../Base.t.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+
+contract EmailRecoveryModule_processRecovery_Test is EmailRecoveryModuleBase {
+    using ModuleKitHelpers for *;
+    using Strings for uint256;
+
+    string public recoveryDataHashString;
+    bytes[] public commandParams;
+    bytes32 public nullifier;
+
+    address owner = vm.addr(2);
+    address nonOwner = address(0x2);
```

Comment:
These are the same address?

Reply:
@JohnGuilding 
No, I think vm.addr(2) is the second address of addresses which is generated by default seed, but address(0x2) is just an example address not practical use

```
  owner 0x2B5AD5c4795c026514f8317c7a215E218DcCD6cF
  nonOwner 0x0000000000000000000000000000000000000002
```

Reply:
Ah cool didn't know that. Happy to leave as is